packages/hint-babel-config/tests/is-valid.ts

```diff
@@ -55,7 +55,7 @@ const tests: Array<HintLocalTest> = [
         name: 'If .babelrc contains an invalid extends, it should fail',
         path: path.join(__dirname, 'fixtures', 'invalid-extends'),
         reports: [
-            { message: `Unexpected token ' in JSON at position 191` }
+            { message: `Unexpected token ' in JSON at position 202` }
```

Comment:
@alrra and @molant, I changed this because the position was off on my machine, but it looks like the original value was expected in CI. Upon further review, it seems this position is different when running on Windows. I suspect this is due to Windows using `\r\n` instead of `\n` for newlines as the difference is exactly the same as the number of lines before the token.

I can (1) simply put this back to the old value to pass CI, (2) update the test (and similar tests) to not depend on the exact position, or (3) update how we're loading the test fixture files so we always use `\n`, making the offsets match cross-OS. 

My preference is (3), but I'd like your opinions.

Reply:
Maybe you are checking out your files with `\r\n` in your machine?
What do you think of:

- Make sure those filed don't have `\r\n`
- Change your options to "Checkout as-is, commit Unix-style": `git config --global core.autocrlf input` if [I understand the guide correctly](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_formatting_and_whitespace) although maybe there's a way to enforce this in the project's configuration in `.gitattributes`?

I don't think that should cause any issue with VS Code and will not need any code changes on your side. We might

Reply:
> Don't use eol=lf it will bite users in the future.

Ok, I'll remove it.

I'll have to use something slightly different than `os.EOL` to avoid breaking the test case for @molant who already has user settings to checkout with LF even on Windows...

I can load the actual JSON test file separately and check for `\r\n` to alter the position if that's fine with you.

Reply:
> My recommendation is to not use `eol=lf` as it will bite users in the future.

I'm curious about this. Why do you think this can bite users in the future? All Windows editors (including notepad) support this line ending and not having this option has already bite us.

Reply:
> All Windows editors (including notepad) support this line ending and not having this option has already bite us.

Ok.

Reply:
>  I'll take that as we're keeping `eol=lf`

Yes.